lib/foreman_openscap/engine.rb

```diff
@@ -121,12 +121,14 @@ class Engine < ::Rails::Engine
                      :resource_type => 'SmartProxy'
         end
 
-        role "Compliance viewer", %i[view_arf_reports view_policies view_scap_contents view_tailoring_files view_openscap_proxies]
+        role "Compliance viewer", %i[view_arf_reports view_policies view_scap_contents view_tailoring_files view_openscap_proxies],
+             "TODO"
```

Comment:
While we're on this, is there a plugin DSL that automatically create these two roles? Something like `add_default_roles`. That would make sure we always have all available view permissions. Not sure what description it would generate though. I guess person with this role is probably a compliance auditor or supervisor. It allows access to policy configuration and scan results and allows downloading reports from the system.

Reply:
So the role description could be something like this?
`Role granting all permissions to policy configuration, scan results and downloading reports.`

Reply:
not `all` but `read` or `view`